app/blueprints/assessments/templates/macros/fund_dashboard_summary.html

```diff
@@ -14,15 +14,17 @@
                 </br>
                 </br>
                 <a class="govuk-link" href="{{ feedback_export_href }}">
-                    Export applicants feedback information
+                    Export feedback survey responses
                 </a>
             {% endif %}
         {% endif %}
-        </br>
-        </br>
-        <a class="govuk-link" href="{{ assessment_tracker_href }}">
-            Assessment Tracker Export
-        </a>
+        {% if assessment_tracker_href %}
+            </br>
+            </br>
+            <a class="govuk-link" href="{{ assessment_tracker_href }}">
+                Assessment Tracker Export
```

Comment:
Do we need the export for "Assessment Tracker" @srh-sloan? There's not going to be any scoring for EOI, so doesn't make sense for us to enable this?

Reply:
It won't be - that link isn't passed in if it's EOI